site/content/docs/api-checks/_index.md

```diff
@@ -21,3 +21,7 @@ This is where you determine whether the response of the HTTP request is correct
 [Read more about assertions]({{< ref "assertions.md" >}})
 
 These two phase are executed under the hood by the Node.js request library and the chai.js assertions library. All checks are executed on Amazon's AWS Lambda infrastructure.
+
+{{<info>}}
```

Comment:
@josep-fl it's good to try and surface this stuff, but the API check overview page is not the right place. I'd suggest looking for something where such a specific callout makes more sense. It doesn't make sense to tell everyone who needs any kind of info about API checks that they cannot use dns.setServers.

If there is no such place, I would simply not expose this specific piece of info. Both conditions need to be true at the same time: having a piece of information on our KB that should be public AND having a place in the public-facing docs where it makes sense (from a user's perspective) to put it.

Reply:
@ragog agreed. I couldn't find a place for this info, it feel too specific.